src/figma/tokens.json

```diff
@@ -937,71 +937,77 @@
       }
     },
     "shadows": {
-      "xs": {
+      "XS": {
         "value": {
           "color": "#0000001A",
           "type": "dropShadow",
-          "x": 0,
-          "y": 2,
-          "blur": 4,
-          "spread": 0
+          "x": "0",
+          "y": "2",
+          "blur": "4",
+          "spread": "0"
         },
-        "type": "boxShadow"
+        "type": "boxShadow",
+        "description": "Uses : Cards"
       },
-      "sm": {
+      "SM": {
         "value": {
           "color": "#0000001A",
           "type": "dropShadow",
-          "x": 0,
-          "y": 2,
-          "blur": 8,
-          "spread": 0
+          "x": "0",
+          "y": "2",
+          "blur": "8",
+          "spread": "0"
         },
-        "type": "boxShadow"
+        "type": "boxShadow",
+        "description": "Uses : Dropdown, Menus"
       },
-      "md": {
+      "MD": {
         "value": {
+          "x": "0",
+          "y": "2",
+          "blur": "16",
+          "spread": "0",
           "color": "#0000001A",
-          "type": "dropShadow",
-          "x": 0,
-          "y": 2,
-          "blur": 16,
-          "spread": 0
+          "type": "dropShadow"
         },
-        "type": "boxShadow"
+        "type": "boxShadow",
+        "description": "Uses : Toast"
       },
-      "lg": {
+      "LG": {
         "value": {
+          "x": "0",
+          "y": "2",
+          "blur": "40",
+          "spread": "0",
           "color": "#0000001A",
-          "type": "dropShadow",
-          "x": 0,
-          "y": 8,
-          "blur": 40,
-          "spread": 0
+          "type": "dropShadow"
         },
-        "type": "boxShadow"
+        "type": "boxShadow",
+        "description": "Uses : Modals"
       },
       "primary": {
         "value": {
+          "x": "0",
+          "y": "2",
+          "blur": "8",
+          "spread": "0",
           "color": "#037DD633",
-          "type": "dropShadow",
-          "x": 0,
-          "y": 7,
-          "blur": 42,
-          "spread": 0
+          "type": "dropShadow"
         },
-        "type": "boxShadow"
+        "type": "boxShadow",
+        "description": "Uses : Primary-Button hover state"
       },
-      "error": {
+      "danger": {
```

Comment:
@Akatori-Design Should we leave it as error until we do the v2 naming update? Then we can change it all over in one go or do you think it's ok to have some danger and some error? I think we want to be consistent until we covert all of them over?